capstone/src/main/webapp/style.css

```diff
@@ -43,13 +43,32 @@ button {
 }
 
 .inbox {
-  width: 200px;
+  width: 300px;
   border: 5px solid #000000;
   padding: 30px;
 }
 
 .profile-pic {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
   width: 5%;
-  height: 5%;
-  margin-left: 50%;
 }
+
+.top-navigation {
+  background-color: #333333;
```

Comment:
nit: if 3 character hexadecimal notation is possible please switch. same for below!

https://google.github.io/styleguide/htmlcssguide.html#Hexadecimal_Notation

Reply:
Done! Thanks for pointing this out